ramen/dr-policy.yaml

```diff
@@ -8,3 +8,5 @@ spec:
     - dr1
     - dr2
   schedulingInterval: 1m
+  replicationClassSelector: {}
+  volumeSnapshotClassSelector: {}
```

Comment:
Doesn't it break backward compatibility? 

What happens on existing system with dr policy which does not include empty selectors? Will ramen reject the existing dr policy after an upgrade?

But otherwise this looks good, and I already tested this change using the new dr policy installed by ramenctl.